octopoes/tests/robot/CeleryMonitor.py

```diff
@@ -66,12 +67,19 @@ def _monitor_events(self, celery_app: Celery) -> None:
             self._started.set()
             self._receiver.capture(limit=None, timeout=None, wakeup=True)
 
+    @property
```

Comment:
Just a few remarks:
- I think based on the implementation this is not a typical property, but just a method (calc_x_.., get_sum_by_type, ...)
- `defaultdict(int)` would also suffice, since `int()` gives int 0 as well

Reply:
To make it even fancier:
```
from collections import Counter
from operator import itemgetter
sums = Counter(map(itemgetter("type"), self.tasks.values())
```

Reply:
😻

Reply:
Nice one, thanks. Implemented :)